tests/bluechi_test/test.py

```diff
@@ -97,7 +98,7 @@ def setup(self) -> Tuple[bool, Tuple[BluechiControllerMachine, Dict[str, Bluechi
                 if self.run_with_valgrind:
                     node.enable_valgrind()
 
-                node.exec_run('systemctl start bluechi-agent')
+                SystemCtl(node).start_unit("bluechi-agent")
```

Comment:
And the same for `node`:
```suggestion
                node.systemctl.start_unit("bluechi-agent")
```

Reply:
Done